src/twisted/internet/test/test_base.py

```diff
@@ -155,6 +158,32 @@ def fakeGetHostByName(name):
         result.put(IOError("The I/O was errorful"))
 
 
+    def test_resolverGivenBytes(self):
```

Comment:
...or better yet it should assert it receives `str`, which would ensure it matches the documentation.